src/main/java/org/jenkinsci/plugins/kubernetes/auth/KubernetesAuthConfig.java

```diff
@@ -18,6 +20,9 @@ public class KubernetesAuthConfig {
     private final boolean skipTlsVerify;
 
     public KubernetesAuthConfig(String serverUrl, String caCertificate, boolean skipTlsVerify) {
+        if (FIPS140.useCompliantAlgorithms() && skipTlsVerify && serverUrl.startsWith("https://")) {
+            throw new IllegalArgumentException("Skipping TLS verification is not accepted in FIPS mode.");
```

Comment:
nit: how about extracting this string to a `Message.properties`? it's appearing 3 times

Reply:
@PereBueno I extracted the message in 13db16d0d45fb695bd6308fd5dbca4caf0297a25 but using a constant as there is nothing UI visible.